zerver/models/realms.py

```diff
@@ -1191,9 +1191,6 @@ def get_realm_by_id(realm_id: int) -> Realm:
 def get_realm_with_settings(realm_id: int) -> Realm:
     # Prefetch the following settings:
     # * All the settings that can be set to anonymous groups.
-    # This also prefetches can_access_all_users_group setting,
-    # even when it cannot be set to anonymous groups because
-    # the setting is used when fetching users in the realm.
```

Comment:
And maybe this comment is needed as well.

Reply:
fixed!